### PR TITLE
Get mapContainer from RecordStore

### PR DESCRIPTION
**Pull request by ahmetmircik**
Reduces contention among partition threads for mapContainer

### CODE REVIEW

**Comment by eminn** [comment]
👍

File: hazelcast/src/main/java/com/hazelcast/map/impl/operation/MapOperation.java [hunk 1]
```diff
@@ -126,16 +130,16 @@ protected void evict() {
         recordStore.evictEntries();
     }
 
-    private void getOrCreateRecordStore() {
+    private RecordStore getOrNullRecordStore() {
```
Comment (jerrinot) [inline hazelcast/src/main/java/com/hazelcast/map/impl/operation/MapOperation.java]: `getRecordStoreOrNull` ?

Comment (ahmetmircik) [inline hazelcast/src/main/java/com/hazelcast/map/impl/operation/MapOperation.java]: done.

**Comment by jerrinot** [comment]
:+1:

**Pull request merged**